libs/cart/src/effects/cart-address.effects.ts

```diff
@@ -32,7 +32,7 @@ export class DaffCartAddressEffects<T extends DaffCartAddress, V extends DaffCar
       map(() => this.storage.getCartId()),
       switchMap(cartId => this.driver.update(cartId, action.payload)),
       map((resp: V) => new DaffCartAddressUpdateSuccess(resp)),
-      catchError(error => of(error.name === DaffStorageServiceError.name
+      catchError(error => of(error.code === DaffStorageServiceError.code
```

Comment:
`code` is a class field so you would need to do something like `error.constructor.code`.

Reply:
I'm not 100% certain, but I believe typescript compilation may do this automatically.

I'll test tomorrow and verify.

Reply:
I've swapped this for instanceof checking.